src/Persistence_SQL.php

```diff
@@ -572,10 +573,16 @@ public function action($m, $type, $args = [])
                 $field = is_string($args[1]) ? $m->getElement($args[1]) : $args[1];
                 $this->initQueryConditions($m, $q);
                 $m->hook('initSelectQuery', [$q, $type]);
+
+                $expr = "$fx([])";
+                if ($type == 'fx0') {
+                    $expr = "ifnull($expr, 0)";
```

Comment:
This is wrong. Instead of:
```
$expr = "$fx([])";
if ($type == 'fx0') {
    $expr = "ifnull($expr, 0)";
}
```
should be simply (ifnull should be inside sum):
```
$expr = ($type == 'fx0' ? "$fx(ifnull([]), 0)" : "$fx([])");
```